library/std/src/sync/once_lock.rs

```diff
@@ -528,7 +528,7 @@ impl<T> OnceLock<T> {
 
     /// # Safety
     ///
-    /// The value must be initialized
+    /// The value must be set
```

Comment:
I feel this could be written more clearly as any one of these:
 - This OnceLock must be in the set state
 - This OnceLock must have been set to a value
 - This OnceLock must not be empty/unset